feat(docs): add centralized task configuration page for enterprise (#1160)
kestra-io/docs: content/docs/05.enterprise/08.centralized-task-configuration.md

## Secrets

On the Namespaces page, select the namespace where you want to define the secrets. Go to the `Secrets` tab. You can see all the secrets associated with this namespace on this `Secrets` tab's page. Click on "Add a secret" button on the top right of the page.

![add_secret.png](/docs/enterprise/add_secret.png)

You can now define the secret by providing the Key and the Secret value. Save the secret by clicking on the "Save" button at the bottom.

![create_new_secret.png](/docs/enterprise/create_new_secret.png)

The secret key should now start appearing on the `Secrets` tab's page. You can edit the secret's value or delete the secret by click on the appropriate button towards the right of the secret row. You can refer the secret in the flow by using the Key. Here is an example flow:

```yaml
id: query-mysql
namespace: io.kestra.tests

tasks:
  id: query
  type: "io.kestra.plugin.jdbc.mysql.Query"
  url: jdbc:mysql://localhost:3306/test
  username: root
  password: "{{ secret('MYSQL_PASSWORD') }}"
  sql: select * from employees
  fetchOne: true
```

> Note that you could refer this secret only in the flows defined within the same namespace under which the secret has been defined.

Reviewer: ```suggestion
Make sure to use the secret only in flows defined in the same namespace (or child namespace) as your secret.
```

## Task Defaults

> Task Defaults can also be defined at the namespace level. These task defaults are then applied for all the tasks of the corresponding type defined in the flows under the same namespace.
> 
> On the Namespaces page, select the namespace where you want to define the task defaults. Go to the `Task defaults` tab. You can now start writing the task defaults here, or add to the existing task defaults. Save the changes by clicking on the "Save" button at the bottom of the page.

Reviewer: ```suggestion
Task Defaults can also be defined at the namespace level. These task defaults are then applied for all tasks of the corresponding type defined in the flows under the same namespace.

On the Namespaces page, select the namespace where you want to define the task defaults and navigate to the `Task defaults` tab. You can add the task defaults here and save the changes by clicking on the "Save" button at the bottom of the page.
```

![define_task_defaults.png](/docs/enterprise/define_task_defaults.png)

> You can also refer the secrets defined with the same namespace in the task defaults.
> 
> Now, for the given example of task defaults, while defining the flow within the same namespace, you need not specifically mention the password for the MySQL query task as shown in the following flow:

Reviewer: ```suggestion
You can reference secrets and variables defined with the same namespace in the task defaults.

In the example below, you no longer need to add the `password` property for the MySQL query task as it's defined in your namespace-level `taskDefaults`:
```

```yaml
id: query-mysql
namespace: io.kestra.tests

tasks:
  id: query
  type: "io.kestra.plugin.jdbc.mysql.Query"
  url: jdbc:mysql://localhost:3306/test
  username: root
  sql: select * from employees
  fetchOne: true
```

> The password would be applied from the task defaults defined at the namespace level.

Reviewer: ```suggestion

```

## Variables

> Variables defined at the namespace level can be referred by any flow defined under the same namespace.
> 
> On the Namespaces page, select the namespace where you want to define the variables. Go to the `Variables` tab. You can now define the variables on this page. Save the changes by clicking on the "Save" button at the bottom of the page.
> 
> ![define_variables.png](/docs/enterprise/define_variables.png)
> 
> Here is an example flow where the namespace variable is being used:

Reviewer: ```suggestion
Variables defined at the namespace level can be used in any flow defined under the same namespace using the syntax: `{{ namespace.variable_name }}`.

On the Namespaces page, select the namespace where you want to define the variables. Go to the `Variables` tab. You can now define the variables on this page. Save the changes by clicking the "Save" button at the bottom of the page.

![define_variables.png](/docs/enterprise/define_variables.png)

Here is an example flow where the namespace variable is used:
```
